src/index.tsx

```diff
@@ -101,6 +102,7 @@ export {
     MinimizeButton,
     RestoreButton,
     MaximizeButton,
+    CustomButton,
```

Comment:
Export the props as well, so the users can use the interface if they use the default CustomButton component